chain/chain/src/store.rs

```diff
@@ -216,15 +216,21 @@ pub trait ChainStoreAccess {
     /// so the receipts from these blocks are propagated
     fn get_incoming_receipts_for_shard(
         &self,
-        shard_id: ShardId,
+        epoch_manager: &dyn EpochManagerAdapter,
+        mut shard_id: ShardId,
         mut block_hash: CryptoHash,
         last_chunk_height_included: BlockHeight,
     ) -> Result<Vec<ReceiptProofResponse>, Error> {
+        let _span =
+            tracing::debug_span!(target: "store", "get_incoming_receipts_for_shard").entered();
```

Comment:
`target: "store"` is used in store crate, I think here `target: "chain"` would be more appropriate (applies to the all instances in this function)